lomap/graphgen.py

```diff
@@ -166,6 +172,31 @@ def __init__(self, dbase):
 
         return
 
+    #sliu 07/17 add function to pick the lead compounds for the radial graphing option
+    def pick_lead(self):
+        if (self.dbase.nums() * (self.dbase.nums() - 1)/2) != self.dbase.strict_mtx.size:
```

Comment:
Basically, this line was copied from the function generateInitialSubgraphList() in the previous code. The strict_mtx is an SMatrix object defined in the dbmol.py and the size of this matrix was initialed using shape[0]*(shape[0] - 1)/2. So I think the line here just to ensure that during the calculation the dbase.nums and strict_mtx.size is still consistent with each other.